charts/egeria-presentation/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: egeria-presentation
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
```

Comment:
Need some more content in this chart including maintainer, link back to original repository etc.
Also a description, perhaps
'Egeria presentation (reactUI) only'

Reply:
Fixed in f08a6e4ae027